### ISSUE TITLE
Swiper not working correctly when not on 1st page

### ISSUE THREAD

**Issue by edwinvdpol** [nolimits4web/Framework7#422]
Hi.

I have a single page with multiple .page. When I initiate the swiper, it does not work correctly when it is not in the first page. If I put the swiper on the first page, it will render correctly. If not, and I switch to the second page, it does not work.

Can you look into this?

**Comment by nolimits4web** [comment]
Can you show your HTML?

**Comment by edwinvdpol** [comment]
http://edwinvandepol.nl/test.html

Click on the "Wonddashboard" button on the bottom right.
Below the D3 graphic (which is not displayed as it should be aswell) is the swiper.

When you resize the screen of that page manualy, it will resize propperly.

**Comment by nolimits4web** [comment]
I will fix that, but now you should reinit your swiper using pageReInit event for this page:
```
myApp.onPageReinit('wonddashboard', function () {
  wondSliper.update(true)
})

**Comment by nolimits4web** [comment]
This is fixed in 1.0.3
